src/caselawclient/Client.py

```diff
@@ -894,6 +894,13 @@ def get_version_created_datetime(self, judgment_uri: DocumentURIString) -> datet
             "%Y-%m-%dT%H:%M:%S.%f%z",
         )
 
+    def has_html_representation(self, judgment_uri: DocumentURIString) -> bool:
```

Comment:
want to write tests for the xqy / an integration test rather than just a test that mocks the call to marklogic but I will do that in a specific ticket as we dont have that setup yet. Will add that mocking test though

Reply:
Yeah, we still haven't built a good framework for testing our MarkLogic queries are doing what we actually expect.